---
layout: workshop_main_2day
permalink: /metabolomics_2019
title: Metabolomics 2019
header1: Workshop Pages for Students
header2: Informatics and Statistics for Metabolomics 2019
image: /site_images/CBW_metabolomics_icon.jpg
keywords: metabolome-focused experiments, analyzing metabolomic data, metabolome
description: Course covers many topics ranging from understanding metabolomics technologies, data collection and analysis, using pathway databases, performing pathway analysis, conducting univariate and multivariate statistics, working with metabolomics databases, and exploring chemical databases. 
instructors: David Wishart, Jeff Xia
length: 2 days
---

# Welcome <a id="welcome"></a>

[Welcome](https://drive.google.com/a/bioinformatics.ca/file/d/1jt4Fk3Ls4YqIDFUc89cmYOvE067Y0w6H/view?usp=sharing) to Informatics and Statistics for Metabolomics 2019.  

The course schedule can be found [here](https://bioinformaticsdotca.github.io/metabolomics_2019_schedule).

Meet your faculty [here](https://drive.google.com/a/bioinformatics.ca/file/d/1Re5v3ohp39dfbjBeq2tsHNbNg8eSI2ZK/view?usp=sharing)

**CLASS MATERIALS:** You can download the printed course manual [here]().  

***

# Pre-Workshop Materials <a id="preworkshop"></a>

## Laptop Setup Instructions

Instructions to setup your laptop can be found [here](https://bioinformaticsdotca.github.io/metabolomics_2019_prework).

## Pre-workshop Tutorials

Pre-readings and pre-work can be found here [here](https://bioinformaticsdotca.github.io/metabolomics_2019_prework).  
It is in your best interest to complete these before the workshop.

***

# Class Photo

Coming soon  

*** 

# Day 1 <a id="day1"></a>

##  Welcome

  *<font color="#827e9c">Michelle Brazas</font>* 

##  Module 1: Introduction to Metabolomics 

  *<font color="#827e9c">David Wishart</font>*
  
  [Lecture]()  
  

##  Module 2: Metabolite Identification and Annotation 

  *<font color="#827e9c">David Wishart</font>*
  
  [Lecture]() 
  
  [Lab practical](https://drive.google.com/open?id=1_hAsK0JpgVGuT90MeV2uDWPsfaYE-tFd)   
  
  [Instructions for lab](https://bioinformaticsdotca.github.io/metabolomics_2019_mod2lab)
  
 
1. NMR and <a href="cbw.bayesil.ca">Bayesil</a>.  
    
2. GC-MS and <a href="http://gcms.wishartlab.com/">GCMS</a>.  
  
3. LC-MS and XCMS in R.  If R doesn't work, try [XCMS online](https://xcmsonline.scripps.edu/landing_page.php?pgcontent=mainPage).   
  
  
#### Data Set and Results Files:
  
  
##### NMR:

<ul>
<li> <a href="https://github.com/bioinformaticsdotca/Metabolomics_2017/raw/master/CBW_NMR_Data.zip">CBW_NMR_Data.zip</a> </li>
<li> <a href="https://github.com/bioinformaticsdotca/Metabolomics_2017/raw/master/CBW_NMR_result.zip">CBW_NMR_result.zip</a> </li>
</ul>

* [NMR simple example file](https://github.com/bioinformaticsdotca/Metabolomics_2019/blob/master/NMR_example.zip)  

##### GC-Autofit:

Download [this file](https://drive.google.com/a/bioinformatics.ca/file/d/1ZIPj5jVWYUG-LMB3-19Uy3EI9RLn_IDM/view?usp=sharing)  

##### LC-MS:

[Metadata](https://drive.google.com/open?id=0B8AGit-0OvvcY3NKUGtKR1I2dlE)  

*Raw datasets  (raw.zip files)*  

[CKD raw](https://drive.google.com/open?id=0B8AGit-0OvvcWWxkS2tZZ283QzQ)  
[Control raw](https://drive.google.com/open?id=0B8AGit-0OvvcMzdzQUwtQ3NhRXM)  

*Converted datasets (mzxml.zip files)*  

[CDK mzxml](https://drive.google.com/open?id=0B8AGit-0OvvcM3dpTUxXM2lVams)  
[Control mzxml](https://drive.google.com/open?id=0B8AGit-0OvvcTlp6WE04RC1zY0k)  

*XCMS using R (results files)*    

[Diffreport](https://drive.google.com/open?id=0B8AGit-0OvvcNGp4Z0diRmxYSlE)  
[Peak Table](https://drive.google.com/open?id=0B8AGit-0OvvcVnJwbkI3STQzUlk)  

*XCMS online (results files)*  

[Cleaned Diffreport](https://drive.google.com/open?id=0B8AGit-0OvvcM1pPXzhzaXYwNWM)  
[Normalized Results](https://drive.google.com/open?id=0B8AGit-0OvvcNlRNWWhYYkJOQzg)  
[Unnormalized Results](https://drive.google.com/open?id=0B8AGit-0OvvcMWJkbUg0a25kamc)  
  
  
  
#### Links:
  
 * [MetaboMiner](http://wishart.biology.ualberta.ca/metabominer/) 
 * [rNMR](http://rnmr.nmrfam.wisc.edu/) 
 * [BMRB Peaks Server](http://www.bmrb.wisc.edu/metabolomics/query_metab.php) 
 * [BATMAN](http://batman.r-forge.r-project.org/) 
 * [Bayesil](http://bayesil.ca/) 
 * [Golm Database](http://gmd.mpimp-golm.mpg.de/) 
 * [NIST/AMDIS](http://chemdata.nist.gov/) 
 * [CFM-ID](http://cfmid.wishartlab.com/) 
 * [Metlin](http://metlin.scripps.edu/upload.php/) 
 * [MetFusion](http://msbi.ipb-halle.de/MetFusion/) 
 * [Adduct Table](http://fiehnlab.ucdavis.edu/staff/kind/Metabolomics/MS-Adduct-Calculator/) 
 * [MZedDB](http://maltese.dbs.aber.ac.uk:8888/hrmet/search/genip.php) 
 * [MWTWIN](http://www.alchemistmatt.com/mwtwin.html/) 
 * [HighChem](http://www.highchem.com/formula-generator/) 
 * [7GR Software](http://fiehnlab.ucdavis.edu/projects/Seven_Golden_Rules/Software/) 
 * [MyCompoundID](http://mycompoundid.org/) 
 
  
  
##  Module 3: Databases for Chemical, Spectral, and Biological Data  

*<font color="#827e9c">David Wishart</font>*  

[Lecture]()  

[Lab Practical - From Raw Spectra to Biological Insights using MetaboAnalystR 2.0]()

#### Links:
  
 * [HMDB](http://hmdb.ca/) 
 * [DrugBank](http://drugbank.ca/) 
 * [METLIN](http://metlin.scripps.edu/) 
 * [PubChem](http://pubchem.ncbi.nlm.nih.gov/) 
 * [ChEBI](http://www.ebi.ac.uk/chebi/) 
 * [ChemSpider](http://chemspider.com/) 
 * [SDBS](http://sdbs.db.aist.go.jp/) 
 * [BioMagResBank](http://bmrb.wisc.edu/metabolomics/) 
 * [MMCD](http://mmcd.nmrfam.wisc.edu/) 
 * [MassBank](http://www.massbank.jp/) 
 * [BMRB](http://www.bmrb.wisc.edu/) 
 * [NMRShiftDB](http://www.ebi.ac.uk/nmrshiftdb/) 
 * [SMPDB](http://www.smpdb.ca/) 
 * [KEGG](http://www.genome.jp/kegg/) 
 * [Reactome](http://www.reactome.org/) 
 * [BioCyc](http://biocyc.org/) 


# Day 2 <a id="day2"></a>

##  Module 4: Backgrounder in Statistics 

  *<font color="#827e9c">Jeff Xia</font>*
  
  [Lecture]()  
  

##  Module 5: MetaboAnalyst 

  *<font color="#827e9c">Jeff Xia</font>*
  
  [Lecture]() 
    
  [Lab practical](https://drive.google.com/a/bioinformatics.ca/file/d/1nmLqmrw4RpkdEOme8k0WzhfJL_wcIsRN/view?usp=sharing)  
  
    
#### Data Input:
  
  Critical: before uploading your data, perform a sanity check: 
  
* It is a data table separated by comma (.csv) or tab (.txt); 
* Three types of labels: feature names, sample names and group labels (must directly follow sample names);
* All measurements should be numerical values (empty for missing values); 
* For details and screenshot instructions, [click here](http://www.metaboanalyst.ca/faces/docs/Format.xhtml)

* You may have downloaded the NMR results .zip folder yesterday. You will need the .csv file from the .zip folder. [NMR results.zip](https://github.com/bioinformaticsdotca/Metabolomics_2017/raw/master/CBW_NMR_result.zip) (Bayesil output - need to remove annotations lines starting with #, keep one type of ID - name/HMDB ID)
* [Cleaned GCMS results](https://raw.githubusercontent.com/bioinformaticsdotca/Metabolomics_2017/master/CBW_GCMS_examples_clean_2017.csv) (Auto-GCMS output that has been cleaned for you) 
* [Normalized LCMS csv](https://drive.google.com/open?id=0B8AGit-0OvvcM1pPXzhzaXYwNWM)  
* [Unnormalized LCMS csv](https://raw.githubusercontent.com/bioinformaticsdotca/Metabolomics_2017/master/unnormalised_LCMS.csv)  


  
##  Module 6: Future of Metabolomics 

*<font color="#827e9c">David Wishart</font>*
  
[Lecture]() 
  
***

Thank you for attending the Analysis of Metagenomic Data workshop! Help us make this workshop better by filling out [our survey]()  

***
