---
layout: workshop_main_2day
permalink: /metabolomics_2020
title: Metabolomics 2020
header1: Workshop Pages for Students
header2: Informatics and Statistics for Metabolomics 2020
image: /site_images/CBW_metabolomics_icon.jpg
keywords: metabolome-focused experiments, analyzing metabolomic data, metabolome
description: Course covers many topics ranging from understanding metabolomics technologies, data collection and analysis, using pathway databases, performing pathway analysis, conducting univariate and multivariate statistics, working with metabolomics databases, and exploring chemical databases. 
instructors: David Wishart, Jeff Xia
length: 2 days
---

# Welcome <a id="welcome"></a>

[Welcome] to Informatics and Statistics for Metabolomics 2020.  

The course schedule can be found [here](https://bioinformaticsdotca.github.io//metabolomics_2020_schedule)

Meet your faculty here (comming soon)

 

***

# Pre-Workshop Materials <a id="preworkshop"></a>


## Pre-workshop Lecture

Pre-readings and pre-work can be found here [here](https://bioinformaticsdotca.github.io/metabolomics_2020_prework).  
It is in your best interest to complete these before the workshop.

***

# Class Photo

Coming soon  

*** 

# Day 1 <a id="day1"></a>

##  Welcome

  *<font color="#827e9c">Rachade Hmamouchi</font>* 

##  Module 1: Introduction to Metabolomics 

  *<font color="#827e9c">David Wishart</font>*
  
  [Lecture]()  
  
##  Module 2: Metabolite Identification and Annotation 

  *<font color="#827e9c">David Wishart</font>*
  
  [Lecture]() 
  
  [Lab practical](https://drive.google.com/open?id=1_hAsK0JpgVGuT90MeV2uDWPsfaYE-tFd)   
  
  [Instructions for lab](https://bioinformaticsdotca.github.io/metabolomics_2019_mod2lab)
  
 
1. NMR and <a href="cbw.bayesil.ca">Bayesil</a>.  
    
2. GC-MS and <a href="http://gcms.wishartlab.com/">GC-Autofit</a>.  
  
3. LC-MS and <a href="dev.metaboanalyst.ca">MetaboAnalyst</a>. 
  
  
#### Data Set and Results Files:

##### NMR (Bayesil):

<ul>
<li><a href="https://github.com/bioinformaticsdotca/Metabolomics_2017/raw/master/CBW_NMR_Data.zip">CBW_NMR_Data.zip</a></li>
<li><a href="https://github.com/bioinformaticsdotca/Metabolomics_2017/raw/master/CBW_NMR_result.zip">CBW_NMR_result.zip</a></li>
<li><a href="https://github.com/bioinformaticsdotca/Metabolomics_2019/blob/master/NMR_example.zip">NMR simple example file</a></li>
</ul>

##### GCMS (GC-Autofit):

*Example datasets (mzXML.zip files)*

Download [this file](https://drive.google.com/a/bioinformatics.ca/file/d/1ZIPj5jVWYUG-LMB3-19Uy3EI9RLn_IDM/view?usp=sharing)  

##### LC-MS (MetaboAnalyst):

*Example datasets (mzXML.zip files)*

Download [this file](https://drive.google.com/open?id=0B8AGit-0OvvcM3dpTUxXM2lVams)  

*Spectra processing with MetaboAnalyst (example result files)* 

<ul>
<li> <a href="https://drive.google.com/open?id=10G4Vek0RAOR_Zb1CCGs9imFx1Beipvr_">Peak Table</a> </li>
<li> <a href="https://drive.google.com/open?id=1O6QCeBLJD8DWLuaDUnnVsLOyf8WwRlBw">Input for MS Peaks to Path</a> </li>
</ul>  

#### Links:
  
 * [MetaboMiner](http://wishart.biology.ualberta.ca/metabominer/) 
 * [rNMR](http://rnmr.nmrfam.wisc.edu/) 
 * [BMRB Peaks Server](http://www.bmrb.wisc.edu/metabolomics/query_metab.php) 
 * [BATMAN](http://batman.r-forge.r-project.org/) 
 * [Bayesil](http://bayesil.ca/) 
 * [Golm Database](http://gmd.mpimp-golm.mpg.de/) 
 * [NIST/AMDIS](http://chemdata.nist.gov/) 
 * [CFM-ID](http://cfmid.wishartlab.com/) 
 * [Metlin](http://metlin.scripps.edu/upload.php/) 
 * [MetFusion](http://msbi.ipb-halle.de/MetFusion/) 
 * [Adduct Table](http://fiehnlab.ucdavis.edu/staff/kind/Metabolomics/MS-Adduct-Calculator/) 
 * [MZedDB](http://maltese.dbs.aber.ac.uk:8888/hrmet/search/genip.php) 
 * [MWTWIN](http://www.alchemistmatt.com/mwtwin.html/) 
 * [HighChem](http://www.highchem.com/formula-generator/) 
 * [7GR Software](http://fiehnlab.ucdavis.edu/projects/Seven_Golden_Rules/Software/) 
 * [MyCompoundID](http://mycompoundid.org/) 
 
  
  
##  Module 3: Databases for Chemical, Spectral, and Biological Data  

*<font color="#827e9c">David Wishart</font>*  

[Lecture]()  

#### Links:
  
 * [HMDB](http://hmdb.ca/) 
 * [DrugBank](http://drugbank.ca/) 
 * [METLIN](http://metlin.scripps.edu/) 
 * [PubChem](http://pubchem.ncbi.nlm.nih.gov/) 
 * [ChEBI](http://www.ebi.ac.uk/chebi/) 
 * [ChemSpider](http://chemspider.com/) 
 * [SDBS](http://sdbs.db.aist.go.jp/) 
 * [BioMagResBank](http://bmrb.wisc.edu/metabolomics/) 
 * [MMCD](http://mmcd.nmrfam.wisc.edu/) 
 * [MassBank](http://www.massbank.jp/) 
 * [BMRB](http://www.bmrb.wisc.edu/) 
 * [NMRShiftDB](http://www.ebi.ac.uk/nmrshiftdb/) 
 * [SMPDB](http://www.smpdb.ca/) 
 * [KEGG](http://www.genome.jp/kegg/) 
 * [Reactome](http://www.reactome.org/) 
 * [BioCyc](http://biocyc.org/) 

## (Optional) Lab Practical: Spectral Processing and Functional Analysis with MetaboAnalyst(R)
 
 *<font color="#827e9c">Jeff Xia</font>*  

Using the [lab practical](https://drive.google.com/a/bioinformatics.ca/file/d/1ciH-x6WSBSlJkl6bAgZylp-Iuy9IFhgR/view?usp=sharing) follow Protocol 9 on the resulting peak table to gain functional insights from the untargeted metabolomics data.  
  
 [Full Metadata](https://drive.google.com/open?id=1LK9zmXYoFG7DCPnptp3xAybUavmYqUkl)    
  
*Larger example datasets (mzXML.zip files)* (Optional! Intended only for Monday's evening session)

<li>[Group 1](https://drive.google.com/open?id=1sn4IMZOnhtfr2ve0ZSPgTB7LLQz_hBKL)</li>  
<li>[Group 2](https://drive.google.com/open?id=1B8TGacpxoZWZWl74LhpqY78b9_9aVB9e)</li>  
<li>[Group 3](https://drive.google.com/open?id=1rnOWG8E1fB7K82F9aI3n6jrZPqDxZU5Q)</li>  
<li>[Group 4](https://drive.google.com/open?id=1qkz9ovUoGWkVQ3loLyP8gzqazho1xjox)</li>  
<li>[Group 5](https://drive.google.com/open?id=1C4jyL8PB3iMZHQnGfQRz8txe5a9cBNM-)</li>  
<li>[Group 6](https://drive.google.com/open?id=1hI5cwr5HO3ReCDd9ZJYVzE6AUhX7_eNO)</li>  
<li>[Group 7](https://drive.google.com/open?id=1XByBTHS04Ks4K-OvH2F8wj9ZLsYK7rFb)</li>  
  
  
# Day 2 <a id="day2"></a>

##  Module 4: Backgrounder in Statistics 

  *<font color="#827e9c">Jeff Xia</font>*
  
  [Lecture]()  
  

##  Module 5: MetaboAnalyst 

  *<font color="#827e9c">Jeff Xia</font>*
  
  [Lecture]() 
    
  [Lab practical](https://drive.google.com/a/bioinformatics.ca/file/d/1nmLqmrw4RpkdEOme8k0WzhfJL_wcIsRN/view?usp=sharing)  
  
    
#### Data Input:
  
Critical: Before uploading your data, perform a sanity check: 
  
* Verify that it is a data table separated by commas (.csv) or tabs (.txt); 
* For concentration/peak intensity tables: three types of labels should be present; feature names, sample names and group labels (must directly follow sample names);
* All measurements should be numerical values (empty for missing values); 
* For details and screenshot instructions, [click here](http://www.metaboanalyst.ca/faces/docs/Format.xhtml)

* [Data 1](https://www.metaboanalyst.ca/resources/data/human_cachexia.csv) Metabolomic concentration table of 77 urine samples from cancer patients and healthy controls.  
* [Data 2](https://github.com/jsychong/MetaboAnalystR/blob/master/MetaboAnalystR_2_Supplementary_Data/iHMP2_48_metaboanalyst_input.csv) Peak intensity table of 48 fecal samples from pediatric inflammatory bowel disease patients and healthy controls.
* [Data 3](https://www.metaboanalyst.ca/resources/data/mummichog_ibd.txt) Peak list of 48 fecal samples from pediatric inflammatory bowel disease patients and healthy controls.


  
##  Module 6: Future of Metabolomics 

*<font color="#827e9c">David Wishart</font>*
  
[Lecture]() 
  
***

Thank you for attending the Analysis of Metagenomic Data workshop! Help us make this workshop better by filling out [our survey]()  

***
